applications/crossbar/src/modules/cb_port_requests.erl

```diff
@@ -1127,6 +1139,20 @@ send_ported_notification(Context, Id) ->
 %% @doc
 %% @end
 %%--------------------------------------------------------------------
+-spec send_port_scheduled_notification(cb_context:context(), ne_binary()) -> 'ok'.
+send_port_scheduled_notification(Context, Id) ->
+    Req = [{<<"Account-ID">>, cb_context:account_id(Context)}
+           ,{<<"Authorized-By">>, cb_context:auth_account_id(Context)}
+           ,{<<"Port-Request-ID">>, Id}
+           | wh_api:default_headers(?APP_NAME, ?APP_VERSION)
+          ],
+    whapps_util:amqp_pool_send(Req, fun wapi_notifications:publish_port_scheduled/1).
```

Comment:
This won't wait for a response from teletype for whether the email was sent or not, will always return 'ok', and the API call will succeed. Seems...not right.